### PR TITLE
[special] add ellipe and ellipk

### LINKED ISSUE #71071: Implementation of `scipy.special.ellipe` in PyTorch.
### 🚀 The feature, motivation and pitch

Implement `ellipe(m)` special function in PyTorch. This function is used to calculate the elliptic integral of the second kind.
### Mathematical Expression

The function is defined as:

```math
E(m) = ∫ [1 - m * sin(t)^2]^(1/2) dt
```
Here, `m` is tensor input, and `E(m)` is an output, `m = k^2`

### Support in other libraries

`special.ellipe` is available in [Scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.ellipe.html#scipy-special-ellipe), [Julia](https://specialfunctions.juliamath.org/latest/functions_list/#SpecialFunctions.ellipe) and various other libraries.

### Motivation

Elliptic functions are used in several mathematical applications—case in point: To find the circumference of an ellipse along their major and minor axis, etc.

cc: @mruberry @kshitij12345

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by kshitij12345**
Fixes https://github.com/pytorch/pytorch/issues/71071

TODO:
* [ ] Fix for gradient when input is `0`.

### CODE REVIEW

**Comment by kyoungseoun-chung** [comment]
Have any further developments been made regarding this issue?

I am also in need of `ellipk` and `ellipe` functions.
Based on stdlib.js's [ellipk](https://github.com/stdlib-js/math-base-special-ellipk) and [ellipe](https://github.com/stdlib-js/math-base-special-ellipe), I have implemented both functions using torch.tensor. (Scipy version uses a recursive function approach, which is not appropriate for my use case. I need to use `torch.vmap`.)

In spite of this, the performance is significantly worse than the `scipy` version both on `cpu` and `mps` devices.
It would be great to have native support from the `torch` side.

**Comment by BeigeLiu** [comment]
Any update ?
